Test/Framework/FPointBase.cs

```diff
@@ -0,0 +1,15 @@
+using NUnit.Framework;
+
+namespace MonoGame.Tests.Framework
+{
+    /// <summary>
+    /// Base class for test classes with floating point operations.
+    /// </summary>
+    class FPointBase
+    {
+        protected void Compare(float expected, float source)
+        {
+            Assert.That(expected, Is.EqualTo(source).Using(FloatComparer.Epsilon));
+        }
+    }
```

Comment:
I don't like this.  

First it seems to only be there in some effort to reduce typing... something that is not a concern.

Second it requires all tests that use this to derive from this `FPointBase` class.